Xcode/debug.xcconfig

```diff
@@ -27,3 +27,6 @@ GCC_OPTIMIZATION_LEVEL = 0
 
 // 
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) DEBUG=1
+
+// No need for Universal Binaries in debug.
+ONLY_ACTIVE_ARCH = YES
```

Comment:
why not?

Reply:
It doubles build time to compile for both architectures, and generally one is going to debug on the native architecture (not under Rosetta emulation). 

Note that I've merely moved this existing setting from one file to another. It's not something new I'm enabling/disabling.